src/avif.c

```diff
@@ -908,9 +919,10 @@ avifBool avifCleanApertureBoxConvertCropRect(avifCleanApertureBox * clap,
                                              avifPixelFormat yuvFormat,
                                              avifDiagnostics * diag)
 {
+    (void)yuvFormat;
```

Comment:
We probably should also deprecate this function and add a new version of this function, say `avifCleanApertureBoxFromCropRect`, that doesn't have the `yuvFormat` parameter.